content/aws_marketplace.md

```diff
@@ -16,557 +16,120 @@ product = ["client", "workstation"]
-    to the launch wizard.
-
-2.  If you're using your own license, create and copy a pre-signed link
-    with the AWS command line tools and save it. For example:
-
-    ```bash
-    aws s3 presign yourbucket/delivery.license
-    ```
-
-3.  Configure all fields in the CloudFormation template. Use the
-    pre-signed license URL for the `LicenseUrl` field.
-
-4.  Associate the IAM role for backup access.
-
-5.  Run the CloudFormation template to create the Chef Automate
-    instance.
-
-## Install Chef Workstation
-
-While the Amazon Machine Images (AMI) for Chef Automate is provisioning,
-download and install Chef Workstation. Chef Workstation is a collection
-of tools and libraries that are packaged together to make it easy to
-develop cookbooks and resources for a Chef / Chef Automate environment.
-You'll need this to interact with Chef Automate and Chef Infra Server
-from the command line.
-
-## Configure Chef Automate
-
-After the instance has been provisioned and initial configuration has
-completed (usually 10 to 13 minutes) finish configuring Chef Automate
-and Chef Infra Server.
-
-1.  Access the initial configuration page by loading `/biscotti/setup`
-    route. Build the URL by prepending `https://` and appending
-    `/biscotti/setup` to the IP address or public hostname that was
-    automatically assigned to the instance when the Amazon Machine
-    Images (AMI) was launched. For example,
-    `https://<fqdn>/biscotti/setup`. If you used the BYOL image, the
-    CloudFormation stack will have the setup URL in the `Outputs`
-    section.
-
-    {{< note spaces=4 >}}
-
-    {{% notes_chef_aws_ssl %}}
+Chef Automate is an enterprise platform that allows developers, operations, and security engineers to collaborate on application and infrastructure changes with speed and at scale. Chef Automate provides actionable insights across data centers and cloud providers, wherever your nodes live.
 
-    {{< /note >}}
+Chef Automate is the center of the modern Chef platform, providing users with a single source of truth for infrastructure, security, and application automation. The comprehensive dashboard provides real-time views of your configuration management activity. Chef Automate comes bundled with the latest Chef Infra Server, providing the core tools you need to manage your enterprise infrastructure. Data collection is enabled by default, allowing your nodes to report activity in real-time. This instance is free for 60 days or you can bring your license (BYOL).
 
-2.  Use the AWS console or command line tools to determine the Instance
-    ID of your Chef Automate instance. The instance ID is required for
-    authorization to access the setup page.
-
-3.  Fill out the setup form and submit it.
+Use this instance with Chef Workstation installed on your laptop or a separate AWS instance.
 
-4.  Follow the link and log into Chef Automate.
-
-## Configure the workstation
+{{< workstation >}}
```

Comment:
{{< shortcode_name >}} is for HTML shortcodes 
{{% shortcode_name %}} is for MD shortcodes, we've already got this shortcode from #3203 and it would be better to keep it in markdown.

```suggestion
{{% workstation %}}
```